os-init.sh

```diff
@@ -1,5 +1,21 @@
 #!/bin/bash -e
 
+function patchconfig {
+    local file=$1
+    local patch_content=$2
+    local patch_type=${3:-strategic}
+    local tmpfile=$(mktemp)
+
+    sudo cp $file $tmpfile
+
+    sudo /data/src/github.com/openshift/origin/_output/local/bin/linux/amd64/oc ex config patch \
```

Comment:
Will you be able to work this so that it conditionally picks `oc ex` for 3.8+ and `openshift ex` for eariler versions?

Reply:
> Will you be able to work this so that it conditionally picks oc ex for 3.8+ and openshift ex for eariler versions?

Yes. Just got a little sidetracked. :)

Reply:
Sure, sure. No rush ;)